src/pokeStyles.css

```diff
@@ -1,3 +1,27 @@
-image{
-    width: 100px;
+img{
+    width: 150px;
+}
+.pokemon{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+.pokeBox{
+    margin: 1%;
+    border: solid rgb(236, 191, 191) 2px;
+    background-image: radial-gradient:();
+
+}
+.App{
+    display: grid;
```

Comment:
When using `display: grid` you want to make sure you're also setting the grid templates with by setting columns and rows directly or by setting grid-areas. 

grid-template-rows should be in a format like `(1fr 3fr 1fr)` or however you want them to be separated. Same goes for the grid-template-columns. 

I really like using grid-areas as it's easy to just assign a grid area to a component rather than having to make sure it's edge borders align with my grid-layout.